README.md

```diff
@@ -2,57 +2,72 @@
 
 ### What is it good for?
 
-The goal of the Autoclock RPC ansible playbook is to have you caught up on the Solana blockchain within 15 minutes, assuming you have a capable server and your SSH key ready. It formats/raids/mounts disks, sets up swap, ramdisk (optional), downloads from snapshot and restarts everything. It is currently configured for a Latitude.sh s3.large.x86 (see "Optimal Machine Settings" below), but we hope to adapt it more widely later on. For a more catch-all ansible playbook and in depth guide on RPC's refer to https://github.com/rpcpool/solana-rpc-ansible
+The goal of the Autoclock RPC ansible playbook is to have you caught up on the Solana blockchain within 15 minutes, assuming you have a capable server and your SSH key ready. 
+
+It formats/raids/mounts disks, sets up swap, ramdisk (optional), downloads from snapshot and restarts everything. 
+
+It is currently configured for a Latitude.sh s3.large.x86 (see "Optimal Machine Settings" below), but we hope to adapt it more widely later on. 
+
+For a more catch-all ansible playbook and in depth guide on RPC's refer to https://github.com/rpcpool/solana-rpc-ansible
 
 ### Optimal Machine Settings
 
 - Our Latitude.sh s3.large.x86 server starts with the settings below, which we prefer because:
 
-  - the initial state of the machine is cleaner than others that we have tried
-  - disks are named consistently (nvme01, nvme0n2)
-  - ubuntu installed (preferably ubuntu 20.04, 22.04) - this won't work with centos, etc. since they don't use aptitude by default
-  - the login user being ubuntu helps (all the solana operations are done using the solana user that the ansible playbook creates)
-  - ubuntu is in the sudoer's list
-  - unmounted disks are clean - if your root is on one of partitions and you pass it as an argument, this could be disastrous
+  - The initial state of the machine is cleaner than others that we have tried
+  - Disks are named consistently (nvme01, nvme0n2)
+  - Ubuntu installed (preferably ubuntu 20.04, 22.04) - this won't work with centos, etc. since they don't use aptitude by default
```

Comment:
22.04 may not work as intended due to ondemand.service not existing. I am testing with 20.04 now
![image](https://user-images.githubusercontent.com/96934965/204569911-6b08ddfa-1cda-4516-aed2-f0f10f511d6c.png)

Reply:
It works as expected on Ubuntu 20.04
Will need to detect for the version and use an alternate method.